cfpb/consumer-credit-trends
# Generate correct credit groupings and download icons on CSV file links

This PR does two main things:

- updates the ranges for credit profiles in the generated code
- adds a file download icon using CSS to the CSV source file link

It also removes sample rules from the main LESS file that should have been deleted earlier.

## Additions

- Adds needed HTML tags and link modifiers to `src/static/js/templates/svg.hbs` to show the file download icon

## Removals

- Removes sample rules from `main.less`

## Changes

- Updates `src/static/js/templates/charts.js` and `data/README` to use the correct credit profile groupings

## Testing

`gulp build`, `gulp watch`, and `gulp release`.

- Check that the near prime and prime borrower risk profile charts have the correct credit groups (620 - 659 and 660 - 719 respectively) across all markets
- Make sure that all generated charts have the proper changes in the HTML to display the link modifier.

## Review

- @cfarm 
- @mistergone 

[Preview this PR without the whitespace changes](?w=0)

## Screenshots

After pasting into Wagtail, to get the proper styling:

![screen shot 2016-12-19 at 5 59 43 pm](https://cloud.githubusercontent.com/assets/1183435/21332218/03039ea6-c615-11e6-859c-fa9ef65edbe9.png)

## Checklist
